site/en/blog/origin-trial-troubleshooting/index.md

```diff
@@ -422,16 +421,21 @@ status of the feature you're testing.
 {% endAside %}
 
 
-### The origin trial is available in your region {: #region}
+### The origin trial is available for the current user {: #user}
 
-Not all origin trials are supported in all geographical regions.
+Not all origin trials are available for all users, even if a valid token is provided.
```

Comment:
```suggestion
Some origin trials are unavailable to certain users, even if a valid token is provided.
```

Reply:
Framing this in positive instead of negative language will make it easier to localize and understand.